__tests__/app.test.js

```diff
@@ -187,3 +184,51 @@ describe("POST /api/articles/:article_id/comments", () => {
       })
   })
 })
+
+describe('PATCH /api/articles/:article_id', () => {
+  
+  test('200: Successfully update votes when votes are provided', () => {
+    const updateVotes = { inc_votes: 5 }
+    
+    return request(app)
+      .patch('/api/articles/1')  
+      .send(updateVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.votes).toBeGreaterThan(0)
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          votes: expect.any(Number),
+          title: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+        })
+      })
+  })
+
+  test('400: responds with error if article id is invalid', () => {
+    const updateVotes = { inc_votes: 1 }
+    
+    return request(app)
+      .patch('/api/articles/bananas')
+      .send(updateVotes)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Invalid article id')
+      })
+  })
+
+
+  test('404:Responds with error if article does not exist', () => {
+    const updateVotes = { inc_votes: 5 }
+    
+    return request(app)
+    .patch('/api/articles/999999')  
+      .send(updateVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Article not found');
+      })
+  })
```

Comment:
Nice this is a really good start to error handling, but we also want to test if the value of the body being set is not correct. E.g `{ inc_votes: "banana" }`